yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -397,6 +398,19 @@ func (yb *TargetYugabyteDB) GetNonEmptyTables(tables []sqlname.NameTuple) []sqln
 	return result
 }
 
+func (yb *TargetYugabyteDB) TruncateTables(tables []sqlname.NameTuple) error {
+	tableNames := lo.Map(tables, func(nt sqlname.NameTuple, _ int) string {
+		return nt.ForUserQuery()
+	})
+	commaSeparatedTableNames := strings.Join(tableNames, ", ")
+	query := fmt.Sprintf("TRUNCATE TABLE %s", commaSeparatedTableNames)
```

Comment:
What should be our behavior with dependent tables (FK constraints) and table list?
For ex - if some tables are there in the table-list for which truncate and start-clean is happening but their dependent tables are already imported, this will error out to do cascade but I know we can't do `CASCADE` on our own so what should we expect from users?

Reply:
Yeah, was thinking about it; I think we should use defaults, which is RESTRICT, and so if it fails, it fails, the onus is on the user to truncate manually.  
What do you think?

Reply:
Right, but if they truncate it with cascade by them, they will have to pass the dependent table in the list as well right so that will also get imported, so should we mention that somewhere maybe if we can detect it with error msg else in docs only?

Reply:
IMO, that sort of falls outside the purview of voyager because that is something manually done by the user. If they truncate with cascade, they should know to also pass those extra tables in our table-list. I can't really see how we can communicate this in voyager docs 🤔

Reply:
Yeah I was also just thinking, if it is something we can tell them somehow, so probably one thing we can do is enhance the error msg to also include the db side Details (just to help user as we are doing the truncate for all tables at once.)
```
postgres=> truncate table t_a;
ERROR:  cannot truncate a table referenced in a foreign key constraint
DETAIL:  Table "t_b" references "t_a".
HINT:  Truncate table "t_b" at the same time, or use TRUNCATE ... CASCADE.
postgres=> 

```

Reply:
Ah, that's a good idea, works. 
`
Truncating non-empty tables on target DB : [public.u public.fkt public.tfkt public.test_simple public.te]
failed to truncate tables: run query "TRUNCATE TABLE public.\"u\", public.\"fkt\", public.\"tfkt\", public.\"test_simple\", public.\"te\"" on target "127.0.0.1": ERROR: cannot truncate a table referenced in a foreign key constraint (SQLSTATE 0A000) HINT:"Truncate table \"test_simple_fk\" at the same time, or use TRUNCATE ... CASCADE." DETAIL:"Table \"test_simple_fk\" references \"test_simple\"."`

Reply:
looks good, nice idea.